torchsynth/synth.py

```diff
@@ -124,13 +129,33 @@ def randomize(self, seed: Optional[int] = None):
             # Profile to make sure this isn't too slow?
             mt19937_gen = csprng.create_mt19937_generator(seed)
             for parameter in self.parameters():
+                if isinstance(parameter, ModuleParameter) and parameter.frozen:
```

Comment:
Should we use this isinstance check for everything? Like not do `parameter.data.uniform_(0, 1, generator=mt19937_gen)` without it?

Reply:
It's a good question -- is there a chance that we might have a non-ModuleParameter that we would want to randomize?

Reply:
Let's change it to this:

```
else:
   raise ValueError(f"{param} is not a ModuleParameter")
```

Then we'll know for sure.